modules/administration-guide/pages/configuring-workspaces-nodeselector.adoc

```diff
@@ -11,11 +11,16 @@ This section describes how to configure link:https://kubernetes.io/docs/concepts
 
 .Procedure
 
-{prod-short} uses the `CHE_WORKSPACE_POD_NODE__SELECTOR` environment variable to configure `nodeSelector`. This variable can contain a set of comma-separated `key=value` pairs to form the nodeSelector rule, or `NULL` to disable it.
-
+{prod-short} uses `CheCluster` Custom Resource to configure `nodeSelector`:
+[source,yaml]
 ----
-CHE_WORKSPACE_POD_NODE__SELECTOR=disktype=ssd,cpu=xlarge,[key=value]
+spec:
+  devEnvironments:
+    nodeSelector:
+      key: value
 ----
+This section must contain a set of `key=value` pairs to form the nodeSelector rule.
```

Comment:
@vinokurig please, provide an example

Reply:
The snippet above is already an example. The `key:value` pair is a random pair that must match the corresponding node's labael.